File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    let answer = '';
    
    let tmp = s.split(' ');
    
    for(let i=0; i<tmp.length; i++){
        for(let j=0; j<tmp[i].length; j++){
            if(j%2 == 0)
                answer += tmp[i][j].toUpperCase();
            else
                answer += tmp[i][j].toLowerCase();
        }
        
        if(i < tmp.length-1)
            answer += ' ';
    }
    
    return answer;
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    
    int check = 0;
    for(int i=0; i<s.length(); i++){
        if(s[i] == ' ')
        {
            check = 0;
            answer += ' ';
            continue;
        }
        
        if(check % 2 == 0)
            answer += toupper(s[i]);
        else
            answer += tolower(s[i]);
        
        check++;
    }
    
    return answer;
}